Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <BaseOutputPath>$(MSBuildThisFileDirectory)binaries\</BaseOutputPath>
```

Comment:
Can we use the term `artifacts` instead of `binaries` since technically this governs all build outputs which aren't just the so called "binaries" these days (e.g. NuGet packages)?

Reply:
I hear you. I have changed the Directory.Build.props file to centralize only the NuGet packages output.

Reply:
I have a problem with "artifacts" since it has a specific meaning in VSTS automated builds. How about a folder named "outputpackages"?